cocos/platform/win32/CCDevice-win32.cpp

```diff
@@ -171,16 +173,18 @@ class BitmapDC
             if (fontPath.size() > 0)
             {
                 _curFontPath = fontPath;
-                wchar_t * pwszBuffer = utf8ToUtf16(_curFontPath);
-                if (pwszBuffer)
-                {
-                    if (AddFontResource(pwszBuffer))
+                std::thread([fontPath, wnd = _wnd, this]() {
+                    wchar_t * pwszBuffer = utf8ToUtf16(fontPath);
+                    if (pwszBuffer)
                     {
-                        PostMessage(_wnd, WM_FONTCHANGE, 0, 0);
+                        if (AddFontResource(pwszBuffer))
```

Comment:
Will other codes depend on `AddFontResource`?

Reply:
Yes, this will affect following font size measurement, cause incorrect layout. I will close this PR, and revert similar modification on v3.